TODO

```diff
@@ -727,6 +727,9 @@ Features:
     but much rather a disconnect on success.
   - when breaking cycles drop sysv services first, then services from /run, then from /etc, then from /usr
   - when a bus name of a service disappears from the bus make sure to queue further activation requests
+  - use *_raw API from libselinux
+    https://github.com/systemd/systemd/pull/1107
+    https://github.com/systemd/systemd/pull/2797
```

Comment:
I see no point in adding PR URLs to `TODO` really. And about the topic in general, I don't know what's left in this area. @keszybz ?